code/processes/gateway.q

```diff
@@ -355,6 +355,7 @@ getserversindependent:{[req;att;besteffort]
 
 // execute an asynchronous query
 asyncexecjpts:{[query;servertype;joinfunction;postback;timeout;sync]
+ o:query 1;
```

Comment:
what is this? the variable name isn't descriptive and there's no explanation of why this is happening (it's not used until about 30 lines later)

queries can be sent in many formats to TorQ gw, is this always going to work?

Reply:
o will extract a dictionary of query parameters for the DA API queries.  We'll change the name to be more descriptive and add some comments.

Reply:
Queries will work for all formats as desired - DA API will use newer logic but everything else should use previous TorQ version logic.